### ISSUE TITLE
A11y critical violation in ProviderId form field

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#26156]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

A11y critical violation in ProviderId form field

### Version

22.0.5

### Expected behavior

A11y critical violation in ProviderId form field

### Actual behavior

A11y critical violation in ProviderId form field

### How to Reproduce?

Go to Clients > Client registration > Client details and scan using the deque browser extension.
